src/Service/Router/Router.php

```diff
@@ -104,7 +108,7 @@ public function subscribe(
      *
      * @return Response
      *
-     * @throws Authenticator\Exception\Unauthorized
```

Comment:
This exception can actually be thrown in the line `$auth->authenticate($request);` because this (dispatch) method is not taking care of it.